var/spack/repos/builtin/packages/amdblis/package.py

```diff
@@ -34,6 +34,8 @@ class Amdblis(BlisBase):
 
     maintainers("amd-toolchain-support")
 
+    conflicts("target=aarch64:", msg="AMD blis available only on x86_64")
```

Comment:
What about the other targets? Can't we:
```suggestion
    requires("target=x86_64:", msg="AMD blis available only on x86_64")
```
? Would that work?

Reply:
Thank you for your suggestion. Done.